src/elements/content-sidebar/messages.js

```diff
@@ -72,6 +72,16 @@ const messages = defineMessages({
         defaultMessage: 'Modify General Task',
         description: 'modal title for when editing an existing general task',
     },
+    securityBlockedErrorMessage: {
```

Comment:
Can we prefix these with `boxSign` and move them up with the other sign-related messages? Same for the `id` attributes.